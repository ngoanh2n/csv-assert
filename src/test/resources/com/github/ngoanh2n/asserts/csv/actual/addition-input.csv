id,email,firstname,lastname,age,note
1,account01@ngoanh2n.github.io,"Tester 01",Nguyen,21,
2,account02@ngoanh2n.github.io,"Tester 02",Nguyen,22,
