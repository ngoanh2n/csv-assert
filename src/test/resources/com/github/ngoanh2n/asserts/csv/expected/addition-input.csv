id,email,firstname,lastname,age,note
,account01@ngoanh2n.github.io,"Tester 01",Nguyen,,
